plugins/thunderbolt/fu-thunderbolt-controller.c

```diff
@@ -276,17 +281,8 @@ fu_thunderbolt_controller_setup(FuDevice *device, GError **error)
 			if (!fu_thunderbolt_device_check_authorized(FU_THUNDERBOLT_DEVICE(self),
 								    error))
 				return FALSE;
-
-		} else {
-			g_set_error(error,
-				    FWUPD_ERROR,
-				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "updates are distributed as part of the platform");
-			return FALSE;
+			fu_device_add_instance_id(device, device_id);
 		}
-		fu_device_add_instance_id(device, device_id);
-		if (domain_id != NULL)
-			fu_device_add_instance_id(device, domain_id);
```

Comment:
outside of a chromebook (a random AMD laptop with USB4) this is noisy.  See this in the journal:
```
Loading…                 [*******************                    ]14:05:39.559 FuEngine             no GUIDs for device USB4 host controller [51bbfea095223888b882226729b37f8946f2c7e9]
Loading…                 [************************               ]14:05:39.561 FuEngine             no GUIDs for device USB4 host controller [c325ffd0947e866d295390576f606f7b355cd046]
```

Reply:
Agreed, I've moved the usb4 timer above the controller check in https://github.com/fwupd/fwupd/pull/7206